pkg/publish/ingress.go

```diff
@@ -186,14 +188,21 @@ func Ingress(req router.Request, svc *v1.ServiceInstance) (result []kclient.Obje
 		}
 	}
 
+	dnsSecret := &corev1.Secret{}
+	err = req.Client.Get(req.Ctx, router.Key(system.Namespace, system.DNSSecretName), dnsSecret)
+	if err != nil && !apierrors.IsNotFound(err) {
```

Comment:
Why are we ignoring the not found error? It looks like the code after this assumes that the secret exists, so I feel like that error should be returned if we get it here.

Reply:
The secret won't be there, if Runtime was installed with `--acorn-dns=disabled`, in which case we're ok comparing against an empty string for `acornDNSDomain`